module/forest/vertex.go

```diff
@@ -13,6 +15,12 @@ type Vertex interface {
 	Parent() (flow.Identifier, uint64)
 }
 
+// VertexToString returns a string representation of the vertex.
+func VertexToString(v Vertex) string {
```

Comment:
Shall we just implement `String() string`?

Reply:
The reason I didn't do this is to avoid extending the `Vertex` interface for something that's really just to save some space in the error type's `Error` method. And I wanted the same representation regardless of the implementation.